<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWin</class>
 <widget class="QWidget" name="RegisterWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget 
{
	background-color: rgb(66, 74, 80);
}

QLabel 
{
	font-weight: bold;
	color: rgb(255, 255, 255);
	border: none;
}

QRadioButton 
{
	border: none;
	color: rgb(255, 255, 255);
}

QRadioButton::indicator 
{
	width: 13px;
	height: 13px;
	border-radius: 8px;
	border: 2px solid rgb(0, 255, 255);
}

QRadioButton::indicator:checked 
{
	width: 13px;
	height: 13px;
	border-radius: 8px;
	background-color: rgb(0, 255, 255);
}

QDateEdit 
{
	border: 1px solid #404142;
	border-radius: 12px;
	background-color: rgba(37, 40, 49, 0.9);
	font: 24pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QLineEdit
{
	border: 1px solid #404142;
	border-radius: 12px;
	background-color: rgba(37, 40, 49, 0.9);
	font: 14pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QLineEdit#picFileEdit
{
	width:313;
	height:35;
}

QLineEdit:focus 
{
	/* 当被选中时，边框颜色为蓝色，且加粗 */
	border: 2px solid #7b63e1;
}

QPushButton 
{
	color: rgb(255, 255, 255);
	border: 1px solid rgb(0, 255, 255);
	border-radius: 13px;
}

QPushButton::hover 
{
	background-color: rgb(0, 255, 255);
	color: rgb(0, 0, 0);
}

QPushButton::pressed 
{
	/* 点击时的背景颜色 */
	background-color: rgb(66, 74, 80);
	/* 点击时的边框颜色 */
	border: 1px solid rgb(0, 255, 127);
	
	color: rgb(255, 255, 255);
}

QLabel#headpicLb 
{
	background-image: url(:/images/photo.png);
	background-repeat: no-repeat;/* 防止图片重复显示 */
	background-position: center;/* 图片居中显示 */
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>16</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>姓名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>16</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>性别：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mrb">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>男</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="wrb">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>女</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>16</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>生日：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="birthdayEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>24</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>16</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="addressEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>微软雅黑 Light</family>
              <pointsize>16</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>电话：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="phoneEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="registerBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>注册</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1,1">
        <item>
         <widget class="QLabel" name="headpicLb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="picFileEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="addpicBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>添加头像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="videoswitchBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>打开摄像头</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cameraBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>拍照</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
